images/linux/scripts/installers/docker-moby.sh

```diff
@@ -16,16 +16,23 @@ if ! IsInstalled $docker_package; then
     apt-get remove -y moby-engine moby-cli
     apt-get update
     apt-get install -y moby-engine moby-cli
+    apt-get install --no-install-recommends -y moby-buildx
 else
     echo "Docker ($docker_package) is already installed"
 fi
 
 # Run tests to determine that the software installed as expected
 echo "Testing to make sure that script performed as expected, and basic scenarios work"
+echo "Checking the docker-moby and moby-buildx"
+DOCKER_BUILDX=/usr/libexec/docker/cli-plugins/docker-buildx
 if ! command -v docker; then
     echo "docker was not installed"
     exit 1
+elif ! [ -x "$DOCKER_BUILDX" ]; then
```

Comment:
Why do we need to hardcoded the path if we can test the `docker buildx` command result?

Reply:
```
testAdm2@danurgDocker:~$ command -v docker
/usr/bin/docker

testAdm2@danurgDocker:~$ command -v docker buidx
/usr/bin/docker

testAdm2@danurgDocker:~$ command -v docker buidxxx
/usr/bin/docker
```

command -v doesn't work for docker plugins

Reply:
Why do we need to use the `command` to validate docker buidx? Check the exit code.

Reply:
If you have a best approach - just provide it here without many words

Reply:
! [[ $(docker buildx) ]]